### DISCUSSION TITLE
Surveillance Station

### DISCUSSION THREAD

**Discussion by bernard04522** [community/community#54384] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Good morning,
Can you script for Surveillance Station 9.1.0-8660
Thanks in advance
DSM version: DSM 6.2.4-25556 Update 6
Bernard

**Comment by samus-aran** [comment]
Hi @bernard04522 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's community](https://community.synology.com/enu/forum/3). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
